app/src/main/res/layout/item_list_comics.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:padding="16dp">
+
+    <ImageView
+        android:id="@+id/activityComicsThumbnail"
+        android:layout_width="120dp"
+        android:layout_height="160dp"
```

Comment:
Uma boa pratica é a gente não utilizar valores `hardcode` para definir tamanhos de views sabe, ou até mesmo outras coisas. 

https://medium.com/@veniosg/best-practices-for-happy-android-resources-9445c1b521d6
https://www.androidpro.com.br/blog/design-layout/criando-styles-e-themes-no-android/
https://medium.com/@joannekao/android-working-with-themes-and-styles-18cde717f4d

Reply:
Show. Mas esses comentários de layout vou deixar para corrigir só depois naquela tarefinha específica de layout ;)

Reply:
Boaaaa